From cb53612df639971646839fb0933a9acc30bc1fce Mon Sep 17 00:00:00 2001
From: Meticulus <theonejohnnyd@gmail.com>
Date: Thu, 21 Jul 2016 09:17:34 -0500
Subject: [PATCH] recovery: put samsung modules back in ramfs.

-I used to use an extention in the device tree for this but,
it is not longer working : shmkbootimg.mk

Change-Id: Idb90a5c84347f302af7216c32e2d507bbce45984
---
 core/Makefile | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/core/Makefile b/core/Makefile
index 8ec1d8b..7440134 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -949,6 +949,9 @@ $(recovery_ramdisk): $(MKBOOTFS) $(MINIGZIP) $(RECOVERYIMAGE_EXTRA_DEPS) \
 	@echo -e ${CL_CYN}"Copying baseline ramdisk..."${CL_RST}
 	$(hide) (cd $(PRODUCT_OUT) && tar -cf - $(TARGET_COPY_OUT_ROOT) | (cd $(TARGET_RECOVERY_OUT) && tar -xf -))
 	@echo -e ${CL_CYN}"Modifying ramdisk contents..."${CL_RST}
+	mkdir -p $(TARGET_RECOVERY_ROOT_OUT)/lib/modules
+	cp -f $(TARGET_OUT)/lib/modules/j4fs.ko $(TARGET_RECOVERY_ROOT_OUT)/lib/modules/j4fs.ko
+	cp -f $(TARGET_OUT)/lib/modules/param.ko $(TARGET_RECOVERY_ROOT_OUT)/lib/modules/param.ko
 	$(hide) rm -f $(TARGET_RECOVERY_ROOT_OUT)/init*.rc
 	$(hide) cp -f $(recovery_initrc) $(TARGET_RECOVERY_ROOT_OUT)/
 	$(hide) rm -f $(TARGET_RECOVERY_ROOT_OUT)/sepolicy
-- 
1.9.1

